src/api/application/application.ts

```diff
@@ -212,6 +218,16 @@ export class Application extends Base {
     }
 
     /**
+     * Sets new application's shortcut configuration.
+     * @param { Object } config New application's shortcut configuration.
+     * @return {Promise.<void>}
+     * @tutorial Application.setShortcuts
+     */
+    public setShortcuts(config: ConfigInterface): Promise<void> {
+        return this.wire.sendAction('set-shortcuts', Object.assign({}, this.identity, config)).then(() => undefined);
```

Comment:
• Best to put `Object.assign({}, this.identity, config)` on a separate line and assign it to a variable for better readability
• Is `.then(() => undefined);` needed here?